_testdata/vendor/vendoredPkg/vendored.go

```diff
@@ -0,0 +1 @@
+package vendoredPkg
```

Comment:
You might want this at a different path, or maybe generated in a tmp dir by the tests?

IDK, I chose the easiest way, do not hesitate to move it around